Automattic/woocommerce-payments
# Update the fraud and risk tools banner

Fixes #6149

#### Changes proposed in this Pull Request



Update the Fraud & Risk tools banner component to match the latest design iteration.







#### Testing instructions





* The unit tests should be passing
* Run `npm run wp option add wcpay_fraud_protection_settings_active 1` to enable the feature flag
* Run `npm run wp option update wcpay_frt_discover_banner_settings ''` to reset the discover banner settings
* The component layout should match the Figma design (WIM5w2Y10SCNBFdMLSVftj-fi-3696_268417)

-------------------


**Post merge**
